### ISSUE TITLE
Mobile: Log in + Thank you screens

### ISSUE THREAD

**Issue by marko-srb** [WordPress/wporg-showcase-2022#226]
Can we update these two screens on mobile.

![login-thanks-mobile-screens](https://github.com/WordPress/wporg-showcase-2022/assets/111751264/d3fb0062-87c0-4352-880c-cbfb0044e1eb)

I don't think we need to do anything else then expanding the section itself a bit, and move whole paragraph element more below, plus edit rows for copy — as in screenshot. 

Let me know if any help is needed.

**Comment by ndiego** [comment]
This is what I am seeing on the live site. Did you want to minimize the space between the dots and the text? 

<img width="400" alt="image" src="https://github.com/WordPress/wporg-showcase-2022/assets/4832319/16587bd6-0adb-4c55-ba69-6664af63c1b4">

**Comment by marko-srb** [comment]
Oh, then it must be some bug to my phone maybe. 

When I posted the issue, dots were over the title, apologies for not screenshoting it. At the moment, they are very very close to the title. See screenshot.

![IMG_2255](https://github.com/WordPress/wporg-showcase-2022/assets/111751264/39ab12c5-f13d-405c-949a-a8df51919479)

I don't need more space than what you have in the screenshot @ndiego , its just the matter if it looks the same as in your screenshot, or as in mine here on most mobile devices. Let me check on a few more phones, once I have a chance.

Thanks.

**Comment by renintw** [comment]
https://github.com/WordPress/wporg-showcase-2022/assets/18050944/44a3e994-aa64-4920-92f0-d125f3563d55

If the window width is reduced on a desktop, at most, the text gets very close to the dots. 
However, when switching to different mobile devices, due to the reduced height, overlapping issues occur like on the iPhone SE (as seen at the end of the video).

**Comment by ndiego** [comment]
> But I might need to look into it a bit to understand why this CSS was added in the first place, to ensure that the modification won't break anything elsewhere.

Originally, [the design](https://www.figma.com/file/nK6LcMOZMtRAQdIky3PNIA/Showcase?type=design&node-id=3444-15487&mode=design&t=dtGa3M8RYFqZSxrm-0) had the text overlaying the dots. That's my guess.

**Comment by marko-srb** [comment]
> Screen.Capture.on.2023-10-11.at.00-47-45.mov 
> If the window width is reduced on a desktop, at most, the text gets very close to the dots. However, when switching to different mobile devices, due to the reduced height, overlapping issues occur like on the iPhone SE (as seen at the end of the video).

In my experience, narrowing down the browser size, or Inspect > Mobile, isn't as precise as testing it on the phone itself in most cases. For example, the screenshot I provided is Iphone 13 Pro, it has much larger screen than SE, and still has narrow spacing there. 

> Originally, [the design](https://www.figma.com/file/nK6LcMOZMtRAQdIky3PNIA/Showcase?type=design&node-id=3444-15487&mode=design&t=dtGa3M8RYFqZSxrm-0) had the text overlaying the dots. That's my guess.
Yes. But we gave up on that approach, and I assume that wasn't fully fixed on mobile in the meantime.

—
Spacing 30px, or margin-bottom is system spacing we have under each element on mobile design. On top and bottom its 40px, between all elements is 30px. I think that should stay there. :) 

— 

Overall, I am not sure my comments are helpful, so to conclude: 
Ideal behavior (per latest design edits/feedback requests) is to have a nice space between the dots and the title, if that artificially means adding more padding for that purpose (like 10-20 px), lets do it. The example is first screenshot I added in this issue on blue background (it is currently 40px space between the dots and the title, could be 30px as well)...


Let me know if this helps, or if you have any other questions.
